database/migrations/2015_11_24_160035_create_team_keys_table.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateTeamKeysTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('team_keys', function (Blueprint $table) {
+            $table->increments('id');
+            $table->string('public_key', 2024);
+            $table->string('private_key', 2024);
```

Comment:
That's bits, not characters. An ASCII character in 8-bit ASCII encoding is 8 bits. The arbitrary number was just to be able to hold the full key + new lines + start and end padding. Here is an example key:

```
-----BEGIN PUBLIC KEY-----\r\nMIGfMA0GCSqGSIb3DQEBAQUAA4GNADCBiQKBgQDG0VAsFij7YKd4XwLJXYVkaGpW\r\nQ+UNYqUlmAcwu6jA+jWg3ujMWT0\/5v5FtIpdvRaJl4El0Bkd9x6hIGbw\/y\/OTt8C\r\nQWCXwxgVKnHhmVpov3rGr1u+qGkIHh9hZ0r8p5Wq6CNjXwSxh0TxzfevD9XiL3sh\r\noRHqzesd0mGI7FoYEwIDAQAB\r\n-----END PUBLIC KEY-----
```